exporter/zipkinexporter/config.go

```diff
@@ -24,5 +24,6 @@ type Config struct {
 
 	// The URL to send the Zipkin trace data to (e.g.:
 	// http://some.url:9411/api/v2/spans).
-	URL string `mapstructure:"url"`
+	URL    string `mapstructure:"url"`
+	Binary bool   `mapstructure:"binary"`
```

Comment:
`use-binary` may be clearer.

Reply:
kind of feel like it should be `serialization_format` or `serializer`.  Like does http/1.1 with proto bodies count as binary?

Reply:
I think `use_proto` will be more precise.